docs/src/main/asciidoc/_admin-guide/Galleon_provisioning.adoc

```diff
@@ -165,6 +165,10 @@ Layers that you combine with "use-case tailored" layers to extend the capabiliti
 * _observability_: Support for MicroProfile monitoring and configuration features. 
 Includes Health support (optional), _microprofile-config_ (optional), _microprofile-metrics_ (optional) and _open-tracing_ (optional).
 
+Tools layers:
+
+* _web-console_: Support for HAL web console.
```

Comment:
@hpehl WDYT about wording more like this:

"Support for loading the HAL web console from the /console context on the HTTP management interface. Not required to use a HAL console obtained independently and configured to connect to the server."

Maybe too wordy. Or not wordy enough. :) I'm just wanting to get across that this isn't needed for HAL to work; it just gives you a copy of HAL.

Reply:
@bstansberry 
Sounds good to me. I don't think it's too wordy.